tests/integrations/json-basic/tests/ui_tests.rs

```diff
@@ -0,0 +1,45 @@
+use ui_test::{custom_flags::rustfix::RustfixMode, dependencies::DependencyBuilder, *};
+
+fn main() -> ui_test::color_eyre::Result<()> {
+    let path = "../../../target";
+    let mut config = Config {
+        output_conflict_handling: if std::env::var_os("BLESS").is_some() {
+            ui_test::bless_output_files
+        } else {
+            ui_test::error_on_output_conflict
+        },
+        bless_command: Some("cargo test".to_string()),
+        ..Config::rustc("tests/actual_tests")
```

Comment:
instead of copying the basic test suite, use a path here pointing to the already existing test suite

Reply:
How did I not think of this ... will do!

Reply:
That may just be my whacky test setup. So if there is too much trouble I'll check out your PR locally and fix it myself.

Reply:
I wasn't able to figure out what the source of the auxiliary file build failure is, so I could use your help Oli.

I also noticed something else - the diffing portion of the testing seems to have a character limit. This was cutting off the terminating JSON brace among other things, and I thought that there was a general issue with the escaping. Here is what I observed:

Diff report:
![image](https://github.com/user-attachments/assets/49739c92-4057-4eac-9afe-5f8e0d86486b)

Same "pure" output:
![image](https://github.com/user-attachments/assets/79e2f712-6a34-437d-b4bf-757ec274d3dc)

How do you want me to handle this?